store/map.js

```diff
@@ -42,54 +42,67 @@ export const mutations = {
 };
 
 export const actions = {
-  createMap(ctx, {map}) {
+  createMap(ctx, { map }) {
     const { commit } = ctx;
-    commit(types.CREATE_MAP, {map});
+    commit(types.CREATE_MAP, { map });
```

Comment:
Your action simply commits a mutation.
Do you plan on `createMap` doing multiple things?

Otherwise you're just calling `map/createMap` -> `map/create`... why not just call `map/create`?